packages/cbioportal-clinical-timeline/src/Timeline.tsx

```diff
@@ -203,14 +250,31 @@ const Timeline: React.FunctionComponent<ITimelineProps> = observer(function({
         zoomSelectBox: useRef(null),
         zoomSelectBoxMask: useRef(null),
         cursorText: useRef(null),
+        hoverStyleTag: useRef(null),
+        timelineViewPort: useRef(null),
     };
 
+    const memoizedHoverCallback = useCallback(
+        (e: React.MouseEvent) => {
+            hoverCallback(e, refs.hoverStyleTag);
+        },
+        [refs.hoverStyleTag]
```

Comment:
why does this not break hovering, since you've only listed refs.hoverStyleTag as the dependency, which will never change?

Reply:
not sure what you mean.  the style tag just sits there and we update it's text